theano/configdefaults.py

```diff
@@ -1327,6 +1328,10 @@ def default_blas_ldflags():
         if res:
             return res
 
+        for warn in warn_record:
```

Comment:
This commit seem good as it only silence the warning when we can reuse numpy blas. By to help the maintenance, can you add this comment that explain what we want to do or similar:

    If we are using conda and can't reuse numpy blas, then doing the fallback and test -lblas could give slow computation, so warn about this. But if we can link to numpy blas like with nomkl packages of conda, don't work, it should be a good enough blas.